src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/BrowserManagement.java

```diff
@@ -319,7 +319,7 @@ public String getRemoteCapabilities() {
         // Null returned from jbrowserdriver
         if (getCurrentWebDriver() instanceof RemoteWebDriver
                 && ((RemoteWebDriver) getCurrentWebDriver()).getCapabilities() != null) {
-            System.out.println(getCurrentWebDriver());
+            logging.info(getCurrentWebDriver().toString());
```

Comment:
Could you add more information about this log:
logging.info(String.format("Sitting capabilities for current browser - %s", getCurrentWebDriver().toString()));